en/micro-integrator/docs/references/connectors/mongodb-connector/mongodb-connector-config.md

```diff
@@ -0,0 +1,1774 @@
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Maximum Pool Size
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            The maximum number of connections in the connection pool.
+        </td>
+        <td>
+            100
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Minimum Pool Size
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            The minimum number of connections in the connection pool.
+        </td>
+        <td>
+            0
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Maximum Idle Time MS
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            The maximum number of milliseconds that a connection can remain idle in the pool before being removed and closed.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Wait Queue Multiple
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            A number that the driver multiples the maxPoolSize value to, to provide the maximum number of threads allowed to wait for a connection to become available from the pool. 
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Wait Queue Timeout MS
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            The maximum time in milliseconds that a thread can wait for a connection to become available. 
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            SSL
+        </td>
+        <td>
+            Boolean
+        </td>
+        <td>
+            A boolean to enable or disables TLS/SSL for the connection.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            SSL Invalid Host Names Allowed
+        </td>
+        <td>
+            Boolean
+        </td>
+        <td>
+            User name used to connect with the file server.
+        </td>
+        <td>
+            -
+        </td>
+        <td>
+            No
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Connect Timeout TS
+        </td>
+        <td>
+            Number
+        </td>
+        <td>
+            The time in milliseconds to attempt a connection before timing out. The default is never to timeout, though different drivers might vary.
```

Comment:
"The time in milliseconds for attempting a connection before timing out. For most drivers, the default is to never timeout."

Reply:
```suggestion
            The time in milliseconds for attempting a connection before timing out. For most drivers, the default is to never timeout.
```